### PR TITLE
[test] Update `cuda-samples` check

### PR DESCRIPTION
**Pull request by jjotero**

### CODE REVIEW

File: config/cscs.py [hunk 1]
```diff
@@ -1074,6 +1075,13 @@
                 'PrgEnv-pgi'
             ]
         },
+        {
+            'name': 'PrgEnv-nvidia',
+            'modules': [
+                'PrgEnv-nvidia',
+                'cdt/21.05'
```
Comment (jgphpc) [inline config/cscs.py:1082]:
```suggestion
                
```
use `-m cdt/21.05` or `-m cdt-cuda/21.05` on the command line ?

Comment (jjotero) [reply config/cscs.py]: @jgphpc If I remove this one, my coda-aware MPI checks fail

**Review by jgphpc — APPROVED** [review]

**Review by victorusu — APPROVED** [review]
`lgtm`

**Review by teojgo — CHANGES_REQUESTED** [review]
The only change I would put is some `FIXME` comments so that we make the corresponding changes for daint as well when the systems are in sync.

File: config/cscs.py (hunk 1 above)
Comment (victorusu) [reply config/cscs.py:1082]: Can we add a `#FIXME` on top of this `cdt` entry? 
We should handle this differently in the future...

**Review by victorusu — CHANGES_REQUESTED** [review]
can you add please a `#FIXME`?

**Review by teojgo — APPROVED** [review]

**Review by victorusu — APPROVED** [review]
`lgtm`

**Comment by teojgo** [comment]
@jenkins-cscs retry dom

**Pull request merged**